R/zzz.R

```diff
@@ -3,8 +3,8 @@
 
 loadModule("ernm",TRUE)
 .onLoad <- function(libname, pkgname){
-	.C("initStats")
```

Comment:
I think its required for Rcpp compileAtttributes to work? I saw there is a commit in the LOLOG package that migrated ERNM to use this. This was also causing me a lot of compilation headaches since the current ERNM package breaks on calling compileAttributes.

I will revert and submit a different PR to migrate over to compileAttributes like the LOLOG packages

Reply:
No need to revert. I just didn't understand. If it works we should include the changes and then do the compile attributes migration later.